app/sampl/base_settings.py

```diff
@@ -100,17 +101,50 @@
 
 CRISPY_TEMPLATE_PACK = "bootstrap4"
 
-LOGIN_TO_AWS = False
+# container engine is either "docker" or "singularity"
+CONTAINER_ENGINE = "docker"
```

Comment:
@robbason should this (CONTAINER_ENGINE) be pulled from environment variables?

Reply:
Let's discuss it in the context of the remote scheduler setup. In my opinion, we could go two ways:
1- environment variables all the way
2 - env variables for things that shouldn't be in config files, like hostnames and passwords, and use different django settings modules for settings that should be grouped together. For example, if you have a remote scheduler, container engine should be singularity.